docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -1487,7 +1487,7 @@ You can specify multiple apps by adding additional list elements.
 </Admonition>
 
 <Admonition type="note" title="IAM roles">
-  For AWS database auto-discovery to work, your agent pods will need to use a role which has appropriate IAM permissions as per the [database documentation](../../database-access/guides/rds.mdx#step-36-configure-iam).
+  For AWS database auto-discovery to work, your agent pods will need to use a role which has appropriate IAM permissions as per the [database documentation](../../database-access/guides/rds.mdx#step-47-configure-iam).
```

Comment:
Should we link to the dedicated page, "Database Access AWS IAM Reference" rather than one section of the RDS guide?

Reply:
Suppose we direct users to the reference page. In that case, we might need to explain how the `teleport db configure bootstrap` works (like we do on the guide pages) since they will be left with almost only policy examples. I'm not sure if this page is the correct one to do it, so we might end up requiring the user to visit 2~3 pages to understand how the policies work and how to use the bootstrap command.